qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -299,21 +298,19 @@ def __compose_clifford_seq(
     def __compose_clifford(
         self, left_elem: SequenceElementType, right_elem: SequenceElementType
```

Comment:
Is this correct type hint? For >2Q .compose is called outside the method and likely now everything is integer number.

Reply:
Resolved in 851cf64